### ISSUE TITLE
Use muhlenberg.github.io?

### ISSUE THREAD

**Issue by benburwell** [Muhlenberg/Club-Website#1]
FYI, the muhlenberg.github.io repository provides free hosting at that domain name if you want to use that one?

There's some very minor stuff there already, nothing you couldn't completely overwrite/start over.

@willshiel @johnsoner94 you guys seem to be the ones working on this

**Comment by johnsoner94** [comment]
Thanks for letting us know! How do you host the domain for the Muhlenberg EMT?

**Comment by benburwell** [comment]
Do you mean the web hosting or the DNS name? Web hosting is currently on Heroku though that's changing around a bit right now (moving a bunch of stuff to Github pages). I am using CloudFlare for free DNS hosting, and the domain name comes from DynDNS.

If you put stuff in the muhlenberg.github.io repository you can point a custom domain name to that if you want to. See [benburwell/benburwell.github.io](https://github.com/benburwell/benburwell.github.io).

**Comment by johnsoner94** [comment]
Yeah, that answers my question. Thanks! I was also wondering if you ever tried to contact Muhlenberg to see if they'd be willing to host it or link to on it on their club page?

**Comment by benburwell** [comment]
Nope, never did that. There isn't really much there, it was honestly really just playing around with the GitHub API to display members/repos. But that sounds like something you guys should do now that you're gonna have a legit website :+1:

**Comment by willshiel** [comment]
Hey Ben,

Thanks for the heads up about github pages.  I saw that there is a way
to get a custom domain by editing the cname file.  Have you had any
experience using this?

Will

**Comment by maborowsky** [comment]
Can't we just buy a name on like namecheap and then just redirect it to github?

**Comment by benburwell** [comment]
Yeah for my own website I have a domain name through name cheap and use the CNAME file to point it to github pages. You just need to put whatever the name is in that file and then make a CNAME record. If you're using not the muhlenberg.github.io repository, you need to use the gh-pages branch rather than master.
